ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -58,9 +58,12 @@ class RuntimeWorker(_worker.PeriodicWorkerThread):
     """
 
     FLUSH_INTERVAL = 10
+    SHUTDOWN_TIMEOUT = 2
```

Comment:
Is there a particular reason for the value 2 or is it arbitrary? IIRC the agent writer timeout is also arbitrary, but is arbitrarily 5 lol. 

The only purpose it would serve is that there is enough time to finish processing one last `flush` as it shuts down, correct? In which case 1 second or less might be fine.

Reply:
I think that's a fair point. I arbitrarily picked a number less than 5 but greater than 0. 2 seemed a good candidate to start with. But we can go with 1.